ui/public/images/ripple.svg

```diff
@@ -0,0 +1,16 @@
+<svg width="110" height="42" viewBox="0 0 110 42" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
why is there both a .svg and a .png of ripple? which one are we actually using? if we're only using one please remove the other one